cognite/neat/core/extractors/graph_sheet_to_graph.py

```diff
@@ -0,0 +1,132 @@
+import logging
+import warnings
+
+import pandas as pd
+from rdflib import RDF, Graph, Literal
+
+from cognite.neat.core.data_classes.transformation_rules import TransformationRules
+
+
+def validate_if_graph_capturing_sheet_empty(graph_capturing_sheet: dict[str, pd.DataFrame]):
+    """Validate if the graph capturing sheet is empty
+
+    Parameters
+    ----------
+    graph_capturing_sheet : dict[str, pd.DataFrame]
+        Graph capturing sheet
+    """
+    if all(df.empty for df in graph_capturing_sheet.values()):
+        logging.error("Graph capturing sheet is empty! Aborting!")
+        raise ValueError("Graph capturing sheet is empty! Aborting!")
```

Comment:
```suggestion
        msg = "Graph capturing sheet is empty! Aborting!"
        logging.error(msg)
        raise ValueError(msg)
```

See argumentation below.

Reply:
Done